cleaners/biglybt.xml

```diff
@@ -29,17 +32,17 @@
         <action command="delete" search="glob" path="$APPDATA\BiglyBT\active\*.bak"/>
   </option>
   <option id="cache">
-    <label>Cache</label>
-    <description>Delete the cache</description>
+    <label>IPFilter Cache</label>
+    <description>Delete the IPFilter Cache</description>
     <action command="delete" search="file" path="~/.biglybt/ipfilter.cache"/>
     <action command="delete" search="file" path="$APPDATA\BiglyBT\ipfilter.cache"/>
   </option>
   <option id="logs">
     <label>Logs</label>
-    <description>Delete the logs</description>
```

Comment:
"Delete the logs" is already translated in many languages, so it's best to reuse the existing string unless there's an important reason